scripts/documentation/generate-executors-data.ts

```diff
@@ -97,6 +98,7 @@ function generateTemplate(
 
     builder.options
       .sort((a, b) => sortAlphabeticallyFunction(a.name, b.name))
+      .sort((a, b) => sortByBooleanFunction(a.required, b.required))
```

Comment:
While we're in here, should we change line 97 from `## Properties` to `## Options`? The latter is used for generators and the wording makes more sense.

Reply:
Yeah, good point. Fixed now!